package.json

```diff
@@ -53,6 +53,7 @@
     "jss": "^5.5.0",
     "jss-preset-default": "^0.5.0",
     "jss-theme-reactor": "^0.4.1",
+    "jss-vendor-prefixer": "^3.0.0",
```

Comment:
It's supposed to be a dependency of `jss-preset-default`. Why do we need it here?